packages/integrations/cloudflare/README.md

```diff
@@ -38,44 +38,45 @@ export default defineConfig({
 
 ## Options
 
-### Mode
+### `mode`
 
 `mode: "advanced" | "directory"`
 
 default `"advanced"`
 
-Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode which picks up the `_worker.js` in `dist`, or a directory mode where pages will compile the worker out of a functions folder in the project root. For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project.
+Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode which picks up the `_worker.js` in the `dist` folder, and `directory`` mode where pages will compile the worker out of a `functions` folder in the project root.
 
 #### `mode:directory`
```

Comment:
**nit:** I'm still not happy with the format of this sub-header. I already thought about the following, but also don't think they are good. Any other ideas?

- `Directory`
- `Directory mode`

Reply:
I mean, we often don't have sub-headings for individual settings for an option anyway. Would be fine to remove entirely and instead, do bullet points to briefly describe the difference between the two, then whatever text you want re: "Switching to directory mode..." in paragraph form.